llvmcpy/llvm.py

```diff
@@ -257,6 +258,28 @@ def call():
         if failure != 0:
             raise LLVMException({})
         return result""".format(result_type, call(), error_message)
+    elif (return_type.kind == "pointer"
+          and return_type.item.kind == "primitive"
+          and return_type.item.cname == "char"
+          and last_arg and last_arg.kind == "pointer"
+          and last_arg.item.kind == "primitive"
+          and last_arg.item.cname in ("unsigned int", "unsigned long")):
```

Comment:
Maybe we could have a global list of "integer" types such as `signed`, `unsigned`, `long`, `int`, `long long`, `long int`, `unsigned int`, `signed int`...